vendor/knative.dev/pkg/reconciler/testing/sorter.go

```diff
@@ -89,5 +89,5 @@ func (o *ObjectSorter) IndexerForObjectType(obj runtime.Object) cache.Indexer {
 }
 
 func emptyIndexer() cache.Indexer {
-	return cache.NewIndexer(cache.MetaNamespaceKeyFunc, cache.Indexers{})
+	return cache.NewIndexer(cache.MetaNamespaceKeyFunc, cache.Indexers{cache.NamespaceIndex: cache.MetaNamespaceIndexFunc})
```

Comment:
The changes to /vendor will need to have to be serialized into a `.patch` file that is applied from `hack/update-deps.sh` so we don't destroy them.